target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/{
+	compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
+    
+	chosen {
+	bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+
+		wlan5g {
+			label = "cf-wr752ac:green:wlan5g";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wlan2g {
+			label = "cf-wr752ac:blue:wlan2g";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		status {
```

Comment:
Change to `led_status: status {` and add an aliases section as for other devices to specify boot LED etc.

Reply:
Or, if this is a LAN led as the label below indicates, ignore my previous comment and name it appropriately. What's the symbol on the device, lan or status?